### PR TITLE
{bio}[gfbf/2023a] PyBEL v0.15.5

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node608.hydra.os - Linux Rocky Linux 8.10, x86_64, AMD EPYC 9384X 32-Core Processor, Python 3.6.8
See https://gist.github.com/WilleBell/cd52b471f50a5e3f723d05ce89d12942 for a full test report.

File: easybuild/easyconfigs/p/PyBEL/PyBEL-0.15.5-gfbf-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        'checksums': ['af8a9b64b821529aca09ebaf6d8d279100d766f19e90b5059ac6a718ca6dee42'],
+    }),
+    ('requests_file', '2.1.0', {
+        'checksums': ['0f549a3f3b0699415ac04d167e9cb39bccfb730cb832b4d20be3d9867356e658'],
+    }),
+    ('pybel', version, {
+        'checksums': ['c2aabd0ad65b9f205f9ed6eae126d19875c1312f8e35641f4f42bcc1fc573f9e'],
+    }),
+]
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/p/PyBEL/PyBEL-0.15.5-gfbf-2023a.eb:73]: this installs some executables, add e.g. `sanity_check_commands = ["pybel --help"]`?

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen3

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
gracehopper - Linux Rocky Linux 9.3, AArch64, ARM UNKNOWN (neoverse_v2), 1 x NVIDIA NVIDIA GH200 480GB, 550.54.15, Python 3.9.18
See https://gist.github.com/migueldiascosta/af6f6f19cbb79a3044acaa7dbb4cf7a6 for a full test report.

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @WilleBell!

**Pull request merged**